### ISSUE TITLE
[Web] After a click to copy a code set focus back in the search bar and select the entire text

### ISSUE THREAD

**Issue by bjherbison** [ente-io/auth#478]
**Is your feature request related to a problem? Please describe.**
After I copy a code I'm always frustrated because the next action I want takes extra steps. If I want the code for the same app I can just click -- but that's not the typical case. The typical case is I want to clear the search and search for some other account.

**Describe the solution you'd like**
Clicking on an account selects the code. To prepare for the the likely event that the next account needed requires a different search the click should also set focus in the search bar and select the text.

Different scenarios:

- If the same account is needed, or another account that fits the search, the focus doesn't interfere with the click.
- If the search is to be refined a click in the search box removes the select.
- If a new search is needed (the common case in my experience) the user can just typing.
- Or just press Backspace or Delete to clear the search box and remove the search.

**Describe alternatives you've considered**
I considered clearing the search (and search box) but that makes a significant visual change that could confuse the user and make it uncertain what had been copied.

**Comment by drcheap** [comment]
I agree 100% with all the bullet points here -- 99% of the time I use a different code "the next time" and yet this idea still doesn't hinder that 1% either.
